Thredds not starting properly mid-install

When executing 
```
esgsetup --config --minimal-setup --thredds --publish --gateway {host} --thredds-password {password}
```
The following error occurs
```
Each entry has the form 'path_identifier | absolute_directory_path':
Current value is: 

esg_dataroot | /esg/data

Enter lines, or <RETURN> to end
INFO       2018-10-08 08:11:13,734 Reinitializing THREDDS server
Got the error: ESGPublishError("Error reinitializing the THREDDS Data Server: error(111, 'Connection refused')",)
Check that the username/password and reinitialization URLs are correct.
Script exited.
```
Looking into `/usr/local/tomcat/logs/catalina.out` the following error caused the thredds startup to fail.
```
SEVERE [localhost-startStop-1] org.apache.catalina.core.StandardContext.startInternal One or more listeners failed to start. Full details will be found in the appropriate container log file
```
There is only one listener in `/usr/local/tomcat/webapps/thredds/WEB-INF/web.xml`
```
<listener>
    <listener-class>org.springframework.web.context.ContextLoaderListener</listener-class>
  </listener>
```
I am not sure why this would not start. Missing `.jar` file?

I found more details regarding the failure to start thredds, from `/usr/local/tomcat/logs/localhost.{date}.log`
```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping': Invocation of init me
thod failed; nested exception is org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [esg.orp.app.tds.TDSAuthorizer] for bean with name 'restrictedDatasetAuthorizer'
 defined in ServletContext resource [/WEB-INF/applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: esg.orp.app.tds.TDSAuthorizer
...
Caused by: java.lang.ClassNotFoundException: esg.orp.app.tds.TDSAuthorizer
```

--- a/esg_init.py
+++ b/esg_init.py
@@ -36,8 +36,8 @@ def init():
 
     cdat_version = "2.2.0"
 
-    esgcet_version = "3.5.4"
-    publisher_tag = "v3.5.4"
+    esgcet_version = "2.6a0"
+    publisher_tag = "v2.6a0"
 
     esgprep_version="2.8.1"
     cmor_version="3.3.2"

